### PR TITLE
docs: update how to theme components

### PR DESCRIPTION
**Pull request by musale**
Closes https://github.com/microsoftgraph/microsoft-graph-toolkit/issues/2086

### CODE REVIEW

**Comment by musale** [comment]
CC @sebastienlevert @gavinbarron @Mnickii for your review. I don't have assign permissions.

**Comment by Lauragra** [comment]
@musale , if you'rea ready for us to review and/or merge your PRs. please set the **ready for content review** and **ready to merge** labels. Thanks.

File: concepts/toolkit/customize-components/style.md [hunk 1]
```diff
@@ -21,48 +21,53 @@ For more flexibility, consider using [custom templates](./templates.md).
...
+
+Using the [mgt-theme-toggle](../components/theme-toggle.md) component, you can set the theming for the document by placing it anywhere that's convenient for your theming experience. This component will render a toggle that you can use to switch between `light` and `dark` modes.
 
 ```html
-<body class="mgt-light">
-    <!-- light theme will apply to all components in this section -->
+<body>
+    <mgt-theme-toggle></mgt-theme-toggle>
     <header><mgt-login></mgt-login></header>
     <article><mgt-agenda></mgt-agend></article>
```
Comment (Mnickii) [inline concepts/toolkit/customize-components/style.md:34]:
```suggestion
    <article><mgt-agenda></mgt-agenda></article>
```

**Review by sebastienlevert — CHANGES_REQUESTED** [review]
I'd love to see a part of this article to cover the programmatical way to toggle theming without the toggle component. This will be super important for Teams tabs, Office.com apps, etc.

**Comment by musale** [comment]
> @musale , if you'rea ready for us to review and/or merge your PRs. please set the **ready for content review** and **ready to merge** labels. Thanks.

@Lauragra I'm unable to edit the labels. Could you grant me the permissions?

**Comment by musale** [comment]
> I'd love to see a part of this article to cover the programmatical way to toggle theming without the toggle component. This will be super important for Teams tabs, Office.com apps, etc.

I have added this @sebastienlevert , you can check it out :)

File: concepts/toolkit/customize-components/style.md [hunk 2]
```diff
@@ -21,48 +21,106 @@ For more flexibility, consider using [custom templates](./templates.md).
 
 ## Apply themes
 
-Two themes are available - light and dark. By default, all components are styled with light theme. To switch to dark theme, you can simply apply `class="mgt-dark"` to the  section of your HTML page. The components inside that section will have dark theme applied. The following examples show how themes will apply based on how you structure your HTML.
+Two themes are available - `light` and `dark`. These themes are supported out of the box because the mgt components are upgraded to use [Fluent UI web elements](/fluent-ui/web-components/) under the hood. By default, all components are in the `light` theme. To switch to `dark` theme, you can do it globally for the entire document using the `mgt-theme-toggle` component or you can theme a component by setting the custom CSS token values on the component.
```
Comment (gavinbarron) [inline concepts/toolkit/customize-components/style.md:24]: We also have teams-light and teams-dark options available for code based theming. 
However I'm pretty comfortable not documenting those yet as we need to do work to align them better with the Teams themes. 
@sebastienlevert thoughts on this?

Comment (sebastienlevert) [reply concepts/toolkit/customize-components/style.md:24]: It seems we support "default", "light", "dark" and "contrast" today. I'd document only "light" and "dark" until we have a stronger Teams story here.

**Review by sebastienlevert — APPROVED** [review]

**Review by Lauragra — APPROVED** [review]

**Pull request merged**